src/hooks/todayTile/useTodayTasks.tsx

```diff
@@ -0,0 +1,126 @@
+}
+
+const useConsentTasks = () => {
+  const { activeProject } = useActiveProject();
+  const { accountHeaders } = useActiveAccount();
+
+  return useRestQuery(
+    'GET /v1/consent/directives/me',
+    {
+      projectId: activeProject?.id!,
+      includeForm: true,
+    },
+    {
+      enabled: !!accountHeaders && !!activeProject,
+      axios: { headers: accountHeaders },
+      select: (data) => data.items,
+    },
+  );
+};
+
+const useGetSurveyResponsesForProject = () => {
+  const { activeSubjectId, activeProject } = useActiveProject();
+  const { accountHeaders } = useActiveAccount();
+
+  return useRestQuery(
+    'GET /v1/survey/projects/:projectId/responses',
+    {
+      projectId: activeProject!.id,
+      author: activeSubjectId!,
+      patientId: activeSubjectId!,
+      includeSurveyName: true,
+      status: 'in-progress,complete',
+      pageSize: 100,
+    },
+    {
+      enabled: !!accountHeaders && !!activeProject && !!activeSubjectId,
+      axios: { headers: accountHeaders },
+      select: (data) => data.items,
+    },
+  );
+};
+
+type GetIncompleteActivitiesCountQueryResponse = {
+  __typename: 'Query';
+  getIncompleteActivitiesCount: number;
+};
+
+const GetIncompleteActivitiesCount = gql`
+  query GetIncompleteActivitiesCount {
+    getIncompleteActivitiesCount
+  }
+`;
+
+export const useGetIncompleteActivitiesCount = () => {
+  const { graphQLClient } = useGraphQLClient();
+  const { isFetched, accountHeaders } = useActiveAccount();
+
+  const queryForPostDetails = useCallback(async () => {
+    return graphQLClient.request<GetIncompleteActivitiesCountQueryResponse>(
+      GetIncompleteActivitiesCount,
+      undefined,
+      accountHeaders,
+    );
+  }, [accountHeaders, graphQLClient]);
+
+  return useQuery(['getIncompleteActivitiesCount'], queryForPostDetails, {
+    enabled: isFetched && !!accountHeaders,
+  });
+};
+
+export const useTodayTasks = () => {
+  const {
+    data: consentTasks,
+    isInitialLoading: loadingConsents,
+    refetch: refetchConsents,
+  } = useConsentTasks();
+  const {
+    data: surveyTasks,
+    isInitialLoading: loadingSurveys,
+    refetch: refetchSurveys,
+  } = useGetSurveyResponsesForProject();
+  const { data, isInitialLoading, refetch } = useGetIncompleteActivitiesCount();
+  const finishedConsentTasks =
+    consentTasks?.filter(
+      (task) => task.status === 'active' || task.status === 'rejected',
+    ) ?? [];
+  const finishedSurveyTasks =
+    surveyTasks?.filter((task) => task.status === 'completed') ?? [];
+  const doneTasks = [
+    ...finishedConsentTasks,
+    ...finishedSurveyTasks,
+  ] as TodayTask[];
+
+  const inProgressConsentTasks =
+    consentTasks?.filter((task) => task.status === 'proposed') ?? [];
+  const inProgressSurveyTasks =
+    surveyTasks?.filter((task) => task.status === 'in-progress') ?? [];
+  const newTasks = [
+    ...inProgressConsentTasks,
+    ...inProgressSurveyTasks,
+  ] as TodayTask[];
+
+  return {
+    doneTasks,
```

Comment:
Is this used?

Reply:
No, we don't have a use for this yet, I've remove it.